docs/content/blog/new-template-delimiter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "A more YAML friendly template delimiter"
+description: "Porter is changing its template delimiter to ${ } to work better with YAML"
+date: "2022-07-26"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://twitter.com/carolynvs"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["v1"]
+---
+
+Porter is getting a different template delimiter in v1!
+Previously, the template delimiter used in porter.yaml was {{ }} and we are changing it to ${ } starting in v1.0.0-beta.2.
+<!--more-->
+
+## What does this change?
+
+The new delimiter avoids the problem of using curly braces in YAML, which forced all templates to be wrapped in quotes.
```

Comment:
I find it a bit confusing that we talks about the problem we are trying to solve but mentions the new delimiter here.
What do you think if we said something like, "The problem with the old delimiter is that it forces all templates to be wrapped in quotes. The quotes makes it impossible to ..."

Reply:
Fixed!